src/utils/spacing.js

```diff
@@ -0,0 +1,65 @@
+import { getWordData, updateUserWordList } from './storage';
+import { getTargetLemma } from '../api/getters';
```

Comment:
Is `spacing` still an appropriate name for this utility file? To me a lot of the stuff in here feels like it belongs to the flash card/quiz domain (maybe a `domains/quiz.js` file would make sense?)